src/test/java/core/basesyntax/FruitShopTest.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.storage.FruitShop;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class FruitShopTest {
+    @BeforeEach
```

Comment:
clear storage in `AfterEach` method
same comment for other tests
```suggestion
    @AfterEach
```